br/pkg/stream/rewrite_meta_rawkv.go

```diff
@@ -282,6 +287,48 @@ func (sr *SchemasReplace) rewriteKVEntryForTable(e *kv.Entry, cf string) (*kv.En
 	return &kv.Entry{Key: newKey, Value: newValue}, nil
 }
 
+func (sr *SchemasReplace) rewriteKVEntryForAutoTableIDKey(e *kv.Entry, cf string) (*kv.Entry, error) {
+	newKey, needWrite, _, err := sr.rewriteKeyForTable(
```

Comment:
needWrite seems always be true, if err is nil？

Reply:
This variable is used to filter table.
If the table need ignore, the needWrite is a false.